packages/circuit-ui/components/Modal/Modal.module.css

```diff
@@ -18,111 +35,75 @@
   );
 }
 
-@media (max-width: 479px) {
-  .base {
-    right: 0;
-    bottom: 0;
-    left: 0;
-    transition: transform var(--cui-transitions-default);
-    transform: translateY(100%);
-  }
-
-  .base::after {
-    height: var(--cui-spacings-mega);
-  }
-}
-
 @media (min-width: 480px) {
-  .base {
-    top: 50%;
-    left: 50%;
-    border-radius: var(--cui-border-radius-mega);
-    opacity: 0;
-    transition: opacity var(--cui-transitions-slow);
-    transform: translate(-50%, -50%);
-  }
-
-  .base::after {
+  .dialog::after {
     height: var(--cui-spacings-giga);
     border-bottom-right-radius: var(--cui-border-radius-mega);
     border-bottom-left-radius: var(--cui-border-radius-mega);
   }
 }
 
-/* Variants */
-
 @media (max-width: 479px) {
-  .contextual {
-    border-top-left-radius: var(--cui-border-radius-mega);
-    border-top-right-radius: var(--cui-border-radius-mega);
+  .dialog {
+    top: unset;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    max-width: 100%;
+    border-bottom-right-radius: 0;
+    border-bottom-left-radius: 0;
```

Comment:
Adding the desktop styles to the unscoped base styles forces you to override the styles here (and hope that you didn't forget any overrides). I find the previous approach that intentionally split and scoped the styles using media queries cleaner and less prone to future bugs.